lib/extensions/firestore_extensions.dart

```diff
@@ -106,18 +116,26 @@ extension ConnectAndConvert on FirebaseFirestore {
         }
         controller.add(StoreProfilePics(profilePics: picIds.toBuiltList()));
       } catch (error, trace) {
-        controller.add(AddProblem.from(
-          message: error.toString(),
-          type: ProblemType.observeProfilePics,
-          traceString: trace.toString(),
-        ));
+        controller.add(
+          AddProblem(
+            problem: ObserveProfilePicsProblem.by(
```

Comment:
In the case of these databse "Observe" events, I'm wondering if we might have a single "ObserveDatabase" event and a corresponding "ObserveDatabaseProblem", which would take the string/enum that defines the database section.  For a future PR I'd say.

Reply:
I had a dig and we already have #184.  I've added a note on also changing to a single "ObserveDatabaseProblem".

Reply:
Good thinking